interface/forms/fee_sheet/review/fee_sheet_ajax.php

```diff
@@ -82,7 +87,16 @@
     }
 
     foreach ($json_procs as $proc) {
-        $procs[] = new procedure($proc->{'code'}, $proc->{'code_type'}, $proc->{'description'}, $proc->{'fee'}, $proc->{'justify'}, $proc->{'modifiers'}, $proc->{'units'}, 0);
+        $procs[] = new procedure(
+            $proc->{'code'},
```

Comment:
I really dislike curly brace syntax like this :( Not sure why we still need to use it.

Reply:
Are there any good php linters or prettierjs-esque  packages that would remove the superfluous curly braces?

Reply:
we use squizlabs/php_codesniffer, https://github.com/openemr/openemr/blob/master/CONTRIBUTING.md#dev_tools_tests, which isn't picking this up